src/Validator/Constraints/CalculationModificationMemberValidator.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Validator\Constraints;
+
+use App\Entity\CalculationModification;
+use App\Entity\Member;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class CalculationModificationMemberValidator extends ConstraintValidator
+{
+
+    public function validate($value, Constraint $constraint): void
```

Comment:
### 🤓 Nitpick: `$value` is a bad name.

Maybe some `$calcModification` will be better to understand